src/components/PatientProvider.tsx

```diff
@@ -3,16 +3,19 @@ import { useFHIRClient } from './FHIRClient';
 
 interface PatientProviderProps {
   children: ReactNode;
+  demoPatient?: fhir.Patient;
 }
 
 export const PatientContext = createContext<fhir.Patient | null>(null);
```

Comment:
Seems like the initial patient (`demoPatient`) should be set here when creating the context.

```suggestion
export const PatientContext = createContext<fhir.Patient | null>(demoPatient);
```

Reply:
When creating the context, it won't have access to the `demoPatient` prop since it is outside of the component.

Reply:
good point